dist/ngWebmakerLogin.js

```diff
@@ -1293,13 +1296,16 @@ ngModule.directive('wmJoinWebmaker', [
 
             if (email) {
               $scope.user.email = email;
-              joinController.validateEmail(email);
```

Comment:
I could leave it where it was, and store the result then use it later, but didn't seem necessary.